### PR TITLE
[EDU-5104] add create databases guide journey

### PR DESCRIPTION
**Pull request by gabriel-azion**

### CODE REVIEW

File: src/content/docs/en/pages/store-journey/sql/create-database.mdx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  Edge sql, sql, database, CRUD, API
+namespace: docs_store_journey_sql_create_database
+menu_namespace: storeMenu
+---
+
+This guide demonstrates how to use Azion **Edge SQL** to create a new database using the Azion API.
+
+## Creating a database
+
+Run the following `POST` request in your terminal, replacing `[TOKEN VALUE]` with your [personal token](/en/documentation/products/guides/personal-tokens/) and `name` with the desired database name to create a new database:
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/store-journey/sql/create-database.mdx:16]:
```suggestion
1. Run the following `POST` request in your terminal, replacing `[TOKEN VALUE]` with your [personal token](/en/documentation/products/guides/personal-tokens/) and `name` with the desired database name to create a new database:
```

File: src/content/docs/en/pages/store-journey/sql/create-database.mdx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+```bash
+curl --location 'https://api.azion.com/v4/edge_sql/databases' \
+--header 'Authorization: Token [TOKEN VALUE]' \
+--header 'Content-Type: application/json' \
+--data '{
+   "name": "my-database"
+}'
+```
+
+You should receive the following response:
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/store-journey/sql/create-database.mdx:27]:
```suggestion
2. You should receive the following response:
```

File: src/content/docs/en/pages/store-journey/sql/create-database.mdx [hunk 3]
```diff
@@ -0,0 +1,43 @@
+---
+title: Creating an Edge SQL database
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/store-journey/sql/create-database.mdx:2]: How to create?

**Review by MarianaAguilera — APPROVED** [review]

File: src/content/docs/en/pages/store-journey/sql/create-database.mdx [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+description: >-
+  Learn how to create an Edge SQL database.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/store-journey/sql/create-database.mdx:4]:
```suggestion
  Learn how to create an Edge SQL database with Azion.
```

**Review by MarianaAguilera — APPROVED** [review]

File: src/content/docs/en/pages/store-journey/sql/create-database.mdx (hunk 3 above)
Comment (gabriel-azion) [inline src/content/docs/en/pages/store-journey/sql/create-database.mdx:2]:
```suggestion
title: How to create an Edge SQL database
```

**Review by GabrielAzion — APPROVED** [review]

**Pull request merged**